runtime/integration-tests/src/cases/lp/transfers.rs

```diff
@@ -271,7 +272,9 @@ fn transfer_tranche_tokens_domain_to_local_to_domain<T: Runtime>() {
 				Token::FixedBytes(pool_a_tranche_1_id::<T>().into()),
 				Token::Uint(DOMAIN_EVM.into()),
 				Token::Uint(EVM_DOMAIN_CHAIN_ID.into()),
-				Token::Address(Keyring::TrancheInvestor(2).into()),
+				Token::FixedBytes(
+					AccountId::from(as_h160_32bytes(Keyring::TrancheInvestor(2))).to_raw_vec(),
+				),
```

Comment:
This makes the test pass. We need an account coded in some way that later when obtaining the local representation the extra padding does not change. I think the current account conversions are quite confusing, and we should need to simplify this into the `DomainAddress`: [ref](https://kflabs.slack.com/archives/C05TBFRBL15/p1721802319814229)

Reply:
Absolutely! I am wondering why this test has worked in the past .

Reply:
Previously a `Token::Address` was passed as a parameter instead, which seems to be processed differently